CFP.md

```diff
@@ -0,0 +1,36 @@
+# Call for Participation: Inclusive Web Devroom
+
+<b>We invite you to submit a talk about anything that relates to making the Web a more inclusive space!</b>
+
+Websites and the extended web technologies such as browsers, extensions, etc. are regularly used by everyone, but sometimes people can get left behind if a technology hasn't been designed in a way that is inclusive for _everyone_ and not just a specific group of people. We firmly believe in making the Web a more inclusive space and want to encourage others to do so as well. 
+
+## Possible topics
+
+Some possible topics to inspire your talk: 
+
+- Web accessibility
+- Privacy and Security 
+- Performance
+- Localization and Internationalization 
+- Sustainability
+- ... and more! Any other form of work being done to make a the Web more inclusive is welcome! 
+
+
+Anyone who is working on any open-source assistive technologies, or building tools to enhance privacy or improve localization, or any other open-source public interest technology that makes the web more inclusive and helps focus on digital rights are welcome to submit a talk for this Devroom.
+
+_Not sure if your talk is a good fit?_ We encourage you to submit it anyway! 
```

Comment:
we can remove this if we don't want it

Reply:
Are you talking about line 21? I think we should keep it for sure.

Reply:
I was 😄